crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/generated/0.6.2-failure.yml

```diff
@@ -0,0 +1,42 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ try a.b() {} catch {}                                                            │ 0..21
+
+Errors: # 2 total
+  - >
+    Error: Expected Identifier.
+       ╭─[crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/input.sol:1:12]
+       │
+     1 │ try a.b() {} catch {}
```

Comment:
even though a `NamedArgument` is required inside `{}`, it is matched successfully as a post-fix expression, and thus, failing for expecting the `try` body afterwards.